substrate/bin/node/cli/src/cli.rs

```diff
@@ -41,6 +41,12 @@ pub struct Cli {
 	#[arg(long)]
 	pub no_hardware_benchmarks: bool,
 
+	/// Disable the BEEFY gadget.
+	///
+	/// Currently enabled by default.
+	#[arg(long)]
+	pub no_beefy: bool,
```

Comment:
I don't think we need this, kitchensink can always use BEEFY

Reply:
I don't know what would be better. I saw that there is also the option to disable grandpa in kitchensink and I thought that it would be good to also have `--no-beefy`. Personally I would keep it, but I don't have strong preferences. If you think it's better to remove it, I can remove it.

Reply:
the grandpa one is an old relic which was also planned to be removed (but low on prio list so it never gets done :P)

Reply:
Oh, ok. Removed `-no-beefy` then.